Headlines/src/Controllers/TrendingCardsViewController.swift

```diff
@@ -9,14 +9,23 @@
 import UIKit
 import SDWebImage
 import Crashlytics
+import StoreKit
 
-class TrendingCardsViewController: UICollectionViewController {
+class TrendingCardsViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/91a98daca6ef65643efab9e8764a068100b494dc/Headlines/src/Controllers/TrendingCardsViewController.swift#L14"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 233 lines</td>
    </tr>
  </tbody>
</table>